modules/media/ajax/FileUpload.php

```diff
@@ -88,7 +88,7 @@ function uploadFile()
     $db     = \NDB_Factory::singleton()->database();
     $config = NDB_Config::singleton();
     $user   =& User::singleton();
-    if (!$user->hasPermission('media_write')) {
+    if (!$user->hasPermission('media_read')) {
```

Comment:
@racostas why is uploading being changed to read ?

Reply:
It's looks as typo/bug from my side :) but will retest before